cloudbuild.yaml

```diff
@@ -0,0 +1,22 @@
+steps:
+- id: 'go-get'
+  name: 'gcr.io/cloud-builders/go'
+  entrypoint: go
+  args:
+    - get
+    - -t
+    - -v 
+    - ./...
+
+- id: 'test'
+  name: 'gcr.io/cloud-builders/go'
+  entrypoint: '/bin/sh'
+  env:
+    - CGO_ENABLED=1
```

Comment:
Q: Is this actually needed?

Reply:
Context is that we have it disabled for the main repo so if it's required there is some sort of issue

Reply:
Yeah it's used for the race condition checker